test/compatibility/compatibility_test_for_releases.sh

```diff
@@ -170,6 +170,7 @@ create_configs()
     echo "logging_compression=snappy" >> $file_name # We only built with snappy, force the choice
     echo "rows=1000000" >> $file_name
     echo "salvage=0" >> $file_name                  # Faster runs
+    echo "statistics_log.sources=off" >> $file_name # WT-12710 - Prevent statistics from enabling both 'all' and 'sources'
```

Comment:
Added a bit more description for the change, let me know if I've missed the mark - can revert back to suggested change - 

```suggestion
    echo "statistics_log.sources=off" >> $file_name # Prevent WT-12710